gradle/libs.versions.toml

```diff
@@ -70,10 +68,10 @@ hilt-compiler = { module = "com.google.dagger:hilt-compiler", version.ref = "hil
 hilt-testing = { module = "com.google.dagger:hilt-android-testing", version.ref = "hilt" }
 sandwich = { module = "com.github.skydoves:sandwich-retrofit", version.ref = "sandwich" }
 retrofit = { module = "com.squareup.retrofit2:retrofit", version.ref = "retrofit" }
-retrofit-kotlin-serialization = { group = "com.jakewharton.retrofit", name = "retrofit2-kotlinx-serialization-converter", version.ref = "retrofitKotlinxSerializationJson" }
-okhttp-interceptor = { module = "com.squareup.okhttp3:logging-interceptor", version.ref = "okHttp" }
+retrofit-kotlinSerialization = { module = "com.squareup.retrofit2:converter-kotlinx-serialization", version.ref = "retrofit" }
```

Comment:
retrofit-kotlin-serialization is better in terms of clarity and per the naming convention

Reply:
There are different conventions, one of them is that the dash should only be use to separate sub-components and each component name should be camel-case. But I understand you prefer no caml case at all so I'll revert the changes.

Reply:
Also note that the versions section currently use camel case for some components so it's kind of a mix.

Reply:
yes my bad is suppose to be retrofit-kotlinx-Serialization

Reply:
should I replace all `kotlin-serialization` references to `kotlinx-serialization`?

Reply:
That would be really good